frontend/components/organisms/Table/index.tsx

```diff
@@ -117,12 +123,38 @@ const Table = ({ columns, dataSource }: TableProps) => {
                                                                         </form>
                                                                     </Modal>
                                                                 )}
-                                                            <Link
-                                                                className="text-blue-500 hover:text-blue-700"
-                                                                href="#"
-                                                            >
-                                                                <Icons name="table_delete" />
-                                                            </Link>
+                                                            <div>
+                                                                <Button
+                                                                    usage="toggle-modal"
+                                                                    onClick={() =>
+                                                                        openModal(
+                                                                            `delete-${data[key]}`
+                                                                        )
+                                                                    }
+                                                                >
+                                                                    <Icons name="table_delete" />
+                                                                </Button>
+                                                                {activeModal ===
+                                                                    `delete-${data[key]}` &&
+                                                                    isOpen && (
```

Comment:
I suggest to move this to a render function

Reply:
@kenorenzsun this has been adjusted